pkg/plugin/v3/api.go

```diff
@@ -125,6 +130,8 @@ func (p *createAPIPlugin) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&p.resource.Group, "group", "", "resource Group")
 	fs.StringVar(&p.resource.Version, "version", "", "resource Version")
 	fs.BoolVar(&p.resource.Namespaced, "namespaced", true, "resource is namespaced")
+	fs.StringVar(&p.resource.CRDVersion, "crd-version", defaultCRDVersion,
+		"version of CustomResourceDefinition to scaffold. Options: [v1, v1beta1]")
```

Comment:
It shows confuse for me because one thing is the Kubernetes API that I will use to scaffold whcih is the purpose of this flag other thing is the version of CustomResourceDefinition which would lead to me to GKV --version flag.

Reply:
A CRD is only part of an API in controller terms; if we were only creating a CRD then the command would be named `create crd`. The command is named `create api` because we're creating an API for a resource, not just a CRD. Once again, this is why `--crd-version` makes sense and `--api-version` does not. In fact, the `operator-sdk` uses `--crd-version` and I've heard no complaints.

Reply:
I still think that it might bring confuse but it is fine. I think we agreed on that 👍